test/e2e/servicedns.go

```diff
@@ -90,60 +92,147 @@ var _ = Describe("MultiClusterServiceDNS", func() {
 		common.WaitForObject(tl, namespace, serviceDNS.Name, objectGetter, serviceDNS, framework.PollInterval, wait.ForeverTestTimeout)
 	})
 
-	It("ServiceDNS object status should be updated, when service and endpoint are created in member cluster", func() {
-		namespace := f.TestNamespaceName()
+	Context("When ServiceDNS object is created", func() {
+		const (
+			RecordTypeA     = "A"
+			RecordTypeCNAME = "CNAME"
+			minDNSTTL       = 180
+		)
+		federation := "galactic"
+		dnsZone := "dzone.io"
 
-		By("Create the ServiceDNS object")
-		serviceDNSObj := common.NewServiceDNSObject(baseName, namespace)
-		serviceDNS, err := fedClient.MulticlusterdnsV1alpha1().MultiClusterServiceDNSRecords(namespace).Create(serviceDNSObj)
-		framework.ExpectNoError(err, "Error creating MultiClusterServiceDNS object %v", serviceDNS)
-		name := serviceDNS.Name
+		It("ServiceDNS object status should be updated, when service and endpoint are created in member cluster", func() {
+			namespace := f.TestNamespaceName()
 
-		serviceDNSStatus := dnsv1a1.MultiClusterServiceDNSRecordStatus{DNS: []dnsv1a1.ClusterDNS{}}
+			By("Create the ServiceDNS object")
```

Comment:
(No action required) I think the tense of this may be incorrect?  (e.g. ServiceDNS object status should be updated BY Creating (vs Created) the ServiceDNS object).

Reply:
Updated.